core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -82,6 +83,8 @@ public class UtilControllerService {
 
   @Autowired private ConfigurableApplicationContext context;
 
+  private Boolean isCoralBuilt;
```

Comment:
how abt initialize to false always ?

Reply:
If I leave initialized as null I only check one time for the classpath resource on the very first go after that it will always be the correct value returned without having to recompute.